src/Marten/Internal/OpenTelemetry/MartenTracing.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Marten.Internal.OpenTelemetry;
+
+internal static class MartenTracing
+{
+    // See https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/messaging/ for more information
+    public static string MartenTenantId = "MartenTentantId";
+    public static string MartenCorelationId = "MartenCorelationId";
+
+    internal static ActivitySource ActivitySource { get; } = new(
+        "Marten",
+        typeof(MartenTracing).Assembly.GetName().Version!.ToString());
+
+    public static Activity? StartConnectionActivity(Activity? parentActivity =null, IEnumerable<KeyValuePair<string, object?>>? tags =null)
+    {
+        return StartActivity("connection", parentActivity, tags);
+    }
+
+    public static Activity StartActivity(string spanName, Activity? parentActivity =null, IEnumerable<KeyValuePair<string, object?>>? tags =null, ActivityKind activityKind =ActivityKind.Internal)
+    {
+        return ActivitySource.CreateActivity(spanName, activityKind, parentActivity?.ParentId, tags);
```

Comment:
Should this be
```suggestion
        return ActivitySource.StartActivity(spanName, activityKind, parentActivity?.ParentId, tags);
```
?

Otherwise an explicit `Start` needs to be done on the activity.

Or this method should be called `CreateActivity` too, else it's very misleading.